Prettify the wi-fi reporting and add more description

drewvolz opened the pull request:
Part of #2054

<img width=333 alt="pretty-wifi" src="https://user-images.githubusercontent.com/5240843/34643440-7ab83480-f2e1-11e7-8fe7-9d9db1daa46e.png">

drewvolz committed:
Style changes, description of what this is about

rye commented on source/views/help/wifi.js:
```diff
@@ -80,15 +81,45 @@ export class ReportWifiProblemView extends React.Component<Props, State> {
+            </Description>
+            <Description selectable={true}>
+              We will record your current location and what device you are
+              using, then send it to a server that IT maintains. They are
```
>They are planning to look closely at where people are having issues!

We don't really speak on their behalf; I suppose I would be more neutral?  As in,

>The networking team can then use this information to identify where people are having issues!

rye replied:
Also, we should be clear that this information is anonymous, and we do not collect usernames.

rye replied:
I think it's okay to be slightly more wordy here just to be clear about what we're suggesting.  Alternatively, we could add some of this to the legal section and point people that way.

drewvolz replied:
Again, if we change this, let’s be sure the release notes read the same.

rye commented on source/views/help/wifi.js:
```diff
@@ -80,15 +81,45 @@ export class ReportWifiProblemView extends React.Component<Props, State> {
+              College Wi-Fi networks, please tap the button below.
+            </Description>
+            <Description selectable={true}>
+              We will record your current location and what device you are
```
>and what device you are using

could be something more like

>and some general information about the device you are using

drewvolz replied:
Should we change that in the release notes, too?

hawkrives commented on source/views/help/wifi.js:
```diff
@@ -80,15 +81,45 @@ export class ReportWifiProblemView extends React.Component<Props, State> {
-            onPress={this.start}
-            title="Report"
-          />
+      <View>
```
We don't need this View anymore

hawkrives reviewed (changes requested):
Rather than use a Container inside the Card, I'd rather use the `style={}` prop of Card itself to add the padding.

rye committed:
Write the status out into the Button rather than a footer

This will be more familiar to most people.  Since we're using this as human-readable text, I don't see why we need to have a button that doesn't change.

rye committed:
Merge pull request #2063 from StoDevX/wr-change-button-title

Write the status out into the Button rather than a footer

drewvolz committed:
Wording changes

drewvolz committed:
Nix view and container, add styles to card

drewvolz committed:
Remove dup word
